client/src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { AuthFormComponent } from './auth-form/auth-form.component';
+import { ReactiveFormsModule } from '@angular/forms';
+import { RouterModule, Routes } from '@angular/router';
+import { SharedModule } from '../shared/shared.module';
+
+const routes: Routes = [
+  {
+    path: 'login',
+    component: AuthFormComponent,
+  },
+];
+
+@NgModule({
+  declarations: [AuthFormComponent],
+  imports: [
+    CommonModule,
+    ReactiveFormsModule,
+    RouterModule.forChild(routes),
+    SharedModule,
+  ],
+  exports: [AuthFormComponent],
```

Comment:
In the current state of your code you don't need this export. But I'd recommend to keep the export and move /login route to app routing.